docs/development/best-practices.md

```diff
@@ -18,7 +18,80 @@ Follow these best practices when you're working on our code.
 // istanbul ignore next: can never happen
 ```
 
-### Logging
+## Code simplicity
+
+### Write simple code
+
+Simple code is easy to read and review.
```

Comment:
```suggestion
Simple code is easy to read, review and maintain.
Choose to write verbose and understandable code instead of "clever" code which might take someone a few attempts to understand what it does.
```